config/bolt/config.yaml

```diff
@@ -23,6 +23,14 @@ theme: base-2021
 # format settings, see: https://www.php.net/manual/en/function.date.php
 date_format: 'F j, Y H:i'
 
+# You can set a preference to omit background images on the login screen.
+omit_backgrounds: true
```

Comment:
This should not be set to true by default, and I'm not even sure i want it in at all.